openapi3/validation_options.go

```diff
@@ -5,33 +5,34 @@ import "context"
 // ValidationOption allows the modification of how the OpenAPI document is validated.
 type ValidationOption func(options *ValidationOptions)
 
-// ValidationOptions provide configuration for validating OpenAPI documents.
+// ValidationOptions provides configuration for validating OpenAPI documents.
 type ValidationOptions struct {
-	SchemaFormatValidationEnabled   bool
-	SchemaPatternValidationDisabled bool
-	ExamplesValidationDisabled      bool
+	schemaFormatValidationEnabled                    bool
+	schemaPatternValidationDisabled                  bool
+	examplesValidationDisabled                       bool
```

Comment:
Please stop touching things not related to your PR. I was asking that the fields you introduce be private, the other ones will come later in another PR.